python/paddle/fluid/layers/control_flow.py

```diff
@@ -912,6 +913,71 @@ def _complete(self):
                    "is_test": self.is_test})
 
 
+def while_loop(cond, body, loop_vars, name=None):
+    """
+    This while_loop operator control. Repeat while_loop body until cond return False.
```

Comment:
em ... "This while_loop operator control" makes no sense.

Reply:
Done. Has been modified into "while_loop control flow."

Reply:
"one of control flows"...